src/components/NavBar.js

```diff
@@ -0,0 +1,29 @@
+import { FaArrowAltCircleLeft, FaMicrophone } from 'react-icons/fa';
+import { SlSettings } from 'react-icons/sl';
+import { Link } from 'react-router-dom';
+import MealList from '../containers/MealList';
+import '../styles/NavBar.css';
+import '../styles/MealList.css';
+
+const NavBar = () => (
+  <nav data-testid="nav" className="nav">
+    <Link className="back" to="/" onClick={() => MealList()}>
+      {' '}
+      <FaArrowAltCircleLeft className="icons" />
+      {' '}
+    </Link>
```

Comment:
- [ ] When I click the back button it goes back to the homepage but refreshes the browser. Reloading the browser defeats the purpose of using React for Single-page applications. Kindly fix your back button so that it does not refresh the browser.